index.html

```diff
@@ -270,87 +291,286 @@
+Web serves people is by protecting them in the face of asymmetries of power and this includes
+establishing and enforcing rules to govern the power of data.
+
+The Web is a social and technical system materialised by [=information flows=], and this document
+focuses on [=privacy=] as it applies to information.
+<dfn data-lt="governance">Data governance</dfn> is the system of [=rules=] that regulate [=information flows=]. When
+[=people=] are involved in these [=information flows=], [=data governance=] determines how
+these rules constrain and distribute the power of information between different [=actors=].
+The <dfn>rules</dfn> describe the way in which different [=actors=] may, must,
+or must not produce or [=process=] flows of information from, to, or about other [=actors=]
+([[?GKC-Privacy]], [[?IAD]]). <dfn>Privacy</dfn> is achieved when the governance situation has
+human impact, either by involving [=personal data=] or by exerting action over [=people=], and the
+[=rules=] are followed appropriately. When the governance situation has human impact but the
+[=rules=] are not followed, there is a <dfn>privacy violation</dfn>. Similarly, we say that a
+particular interaction is <dfn data-lt="appropriately">appropriate</dfn> when the [=rules=] are adhered
+to) or <dfn data-lt="inappropriately">inappropriate</dfn> otherwise.
+
+Typically, <dfn>actors</dfn> are [=people=] but they can also be collective endeavours such as
+companies, associations, or political parties (eg. in the case of espionage), which are known as
+[=parties=]. It is important to keep in mind that not all people are equal in how they may resist
+the imposition of unfair [=rules=], some [=people=] are more [=vulnerable=] and therefore in greater
+need of protection. The focus of this document concerns leveraging this power differential against
+people, but the same techniques are also used to leverage the power of information against other
+entities, such as companies or governments.
+
+The [=rules=] that apply vary from [=context=] to [=context=] ([[?Understanding-Privacy]], [[?CI]]): people
+have different expectations of [=privacy=] at work, at a café, or at home. Understanding and
+evaluating a privacy situation is best done by clearly identifying:
+
+* Its [=actors=], which include the subject of the information as well as the sender and recipient
+  of the [=information flow=]. (Note that recipients might not be intentional.)
+* The specific type of data in the [=information flow=].
+* The [=rules=] that are in use in this specific context.
+
+It is important to keep in mind that there are <em>always</em> privacy [=rules=]. There is no
+purportedly neutral situation in which no [=rules=] apply. We must therefore determine which [=rules=]
+best serve human values ([[?ETHICAL-WEB]], [[?Why-Privacy]]).
+
+<dfn data-lt='information'>Information flows</dfn> are relevant to privacy whenever people are
```

Comment:
This isn't a definition of "information flows". I'm not sure we actually need a definition.

Reply:
I went with an actual definition (but indeed better than that thing). I could be convinced otherwise, but given the potential for confusion about personal data I think it's worth doing.